python/ray/runtime_env.py

```diff
@@ -296,6 +296,8 @@ class MyClass:
 
     known_fields: Set[str] = {
         "py_modules",
+        "java_jars",
+        "dependent_jars",
```

Comment:
`dependent_jars` does not appear to be used. It seems to be an attribute of the RuntimeEnv protobuf.